plugins/woocommerce/src/Internal/Logging/RemoteLogger.php

```diff
@@ -178,22 +178,19 @@ protected function should_handle( $level, $message, $context ) {
 			return false;
 		}
 
-		// Ignore logs that are less severe than critical. This is temporary to prevent sending too many logs to the remote logging service. We can consider remove this if the remote logging service can handle more logs.
-		if ( WC_Log_Levels::get_level_severity( $level ) < WC_Log_Levels::get_level_severity( WC_Log_Levels::CRITICAL ) ) {
-			return false;
-		}
-
 		if ( $this->is_third_party_error( (string) $message, (array) $context ) ) {
 			return false;
 		}
 
-		try {
-			// Record fatal error stats.
-			$mc_stats = wc_get_container()->get( McStats::class );
-			$mc_stats->add( 'error', 'critical-errors' );
-			$mc_stats->do_server_side_stats();
-		} catch ( \Throwable $e ) {
-			error_log( 'Warning: Failed to record fatal error stats: ' . $e->getMessage() ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
+		// Record fatal error stats.
+		if ( WC_Log_Levels::get_level_severity( $level ) === WC_Log_Levels::get_level_severity( WC_Log_Levels::CRITICAL ) ) {
```

Comment:
Should this be critical and above? [Severity list](https://woocommerce.github.io/code-reference/files/woocommerce-includes-class-wc-log-levels.html#source-view.44)

Reply:
Good catch!  Fixed in 7bcf7e79ae13e44e06c8bb8ea7601ad6d4370e92.